rails/Gemfile

```diff
@@ -0,0 +1,10 @@
+source 'https://rubygems.org'
+
+gem 'benchmark-ips', github: 'kirs/benchmark-ips', branch: 'quiet-mode'
+gem 'rails', path: '/rails'
```

Comment:
Is it possible to set the gem without a path to the local `rails` gem?

Perhaps something like this? `gem 'rails', :git => 'git://github.com/rails/rails.git', :ref => '4aded'`